.github/workflows/dotnet/pr_checks.yml

```diff
@@ -0,0 +1,62 @@
+name: Build and Coverage
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ "main", "master", "stage", "dev", "release/*"]
+  pull_request:
+    branches: [ "main", "master", "stage", "dev", "release/*" ]
+    
+env:
+  DOTNET_VERSION: 7.0.x
+  COVERAGE_THRESHOLD: ${{vars.COVERAGE_THRESHOLD}}
```

Comment:
I would pass the `COVERAGE_THRESHOLD` and `DOTNET_VERSION` as inputs, just because it will be more flexible and easier to use.

Reply:
Done